handler/tag.go

```diff
@@ -53,3 +55,37 @@ func (t *Tag) CreateTag(c *gin.Context) {
 
 	c.JSON(http.StatusOK, tagPublic)
 }
+
+func (t *Tag) GetAllTags(c *gin.Context) {
+	tagPublicSlice, err := t.tagService.GetAllTags()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"message": "something happen in the server"})
+		return
+	}
+	c.JSON(http.StatusOK, tagPublicSlice)
+}
```

Comment:
Get all tag should be in pagination form.

Returning it all at once would cause a huge work load fro both server and client.

Reply:
Also, I think it should be done in the separate card id from the get a single tag by id, shouldn't it?.

Reply:
I did it on the wrong card. Then I will move this part to another card

Reply:
I just looking into jira. We don't have GetAllTag. Do we need this endpoint?

Reply:
I removed it first and if we still need this endpoint I will move to another branch

Reply:
Okay, I'll create that card then. In this card, let focus on implementing the get tag by ID API instead.

Reply:
If you already done this, Don't forget to resolve this conversation.